packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -669,6 +724,15 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
 
+    function _getDvmTimestamp(Request storage request, uint256 requestTimestamp) internal view returns (uint256) {
```

Comment:
Might be more accurate to name this `_getTimestampForDvmRequest`

Reply:
Because otherwise I would think that this would return the current time of the DVM

Reply:
Done!